lib/components/dropdown-item.vue

```diff
@@ -17,6 +17,7 @@
 
     export default {
         extends: bLink,
```

Comment:
I think we can use a smarter refactoring approach by just extending and adding `dropdown-item` class to bLink.